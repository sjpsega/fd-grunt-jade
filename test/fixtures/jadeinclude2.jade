include include/head.jade
include include/content.jade
.content aaa