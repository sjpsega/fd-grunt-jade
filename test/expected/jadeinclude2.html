<!-- create by jade! -->

<html>
  <head>
    <title>你好啊</title>
  </head>
  <body></body>
</html>
<div class="inner">哈哈
  <div class="txt">测试测试</div>
</div>
<div class="content">aaa</div>